hbase_sql/src/test/java/com/splicemachine/derby/stream/spark/SparkDataSetTest.java

```diff
@@ -15,15 +15,30 @@
 
 package com.splicemachine.derby.stream.spark;
 
+import com.clearspring.analytics.util.Lists;
 import com.splicemachine.db.iapi.sql.execute.ExecRow;
+import com.splicemachine.db.iapi.types.SQLInteger;
+import com.splicemachine.db.impl.sql.execute.ValueRow;
 import com.splicemachine.derby.impl.SpliceSpark;
 import com.splicemachine.derby.stream.AbstractDataSetTest;
+import com.splicemachine.derby.stream.function.LocatedRowToRowFunction;
 import com.splicemachine.derby.stream.iapi.DataSet;
 import com.splicemachine.derby.stream.spark.SparkDataSet;
+import org.apache.spark.sql.Column;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.junit.Ignore;
+import org.junit.Test;
+import static org.apache.spark.sql.functions.*;
```

Comment:
I'd remove these static wildcard imports, I was confused for a few minutes about where broadcast() was defined.